taxi/admin.py

```diff
@@ -0,0 +1,27 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+
+from taxi.models import Car, Manufacturer, Driver
+
+
+@admin.register(Car)
+class CarAdmin(admin.ModelAdmin):
+    list_display = ("manufacturer", "model")
+    list_filter = ("manufacturer",)
+    search_fields = ("model",)
+
+
+@admin.register(Manufacturer)
+class ManufacturerAdmin(admin.ModelAdmin):
+    pass
```

Comment:
or you can just do admin.site.register(Manufacturer) instead of creating an empty admin class